sync-core/src/test/java/com/cloudant/sync/replication/DBWithSlashReplicationTest.java

```diff
@@ -66,7 +66,7 @@ public void replicationPullAndPushDbWithSlash() throws Exception{
         replicator.start();
         Assert.assertEquals(Replicator.State.STARTED, replicator.getState());
 
-        while(replicator.getState() != Replicator.State.COMPLETE) {
+        while(replicator.getState() != Replicator.State.COMPLETE || replicator.getState() == Replicator.State.ERROR) {
```

Comment:
This and the change to `PullReplicatorTest` look like they belong in a separate commit which explains what the change is for.